content/foundation/voting.md

```diff
@@ -121,6 +120,11 @@ lies with the individual serving as release manager. The
 specifics of the process may vary from project to project, but the 'minimum
 quorum of three +1 votes' rule is universal.
 
+Note that there is no implicit +1 from the release manager,
+or from anyone in any ASF vote. Only explicit votes are valid.
+The release manager can vote on package releases as well
+and is encouraged to cast a vote.
```

Comment:
I don't understand "package releases" in this context, does that just mean that they are encouraged to vote on releases? If yes that might be

> The release manager is encouraged to vote on releases, like any reviewer would do.